### ISSUE TITLE
[android] Kotlin compilation error with RN 0.59.0

### ISSUE THREAD

**Issue by thymikee** [Scandit/barcodescanner-sdk-react-native#109]
## Bug report

Compiling `scandit-react-native` on Android RN v0.59.0 results in a Kotlin compilation error:

```
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/BarcodePicker.kt: (22, 1): Class 'BarcodePicker' is not abstract and does not implement abstract base class member @Nonnull protected/*protected and package*/ abstract fun createViewInstance(@Nonnull p0: ThemedReactContext): BarcodePicker defined in com.facebook.react.uimanager.SimpleViewManager
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/BarcodePicker.kt: (61, 5): 'receiveCommand' overrides nothing
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/BarcodePicker.kt: (86, 5): 'createViewInstance' overrides nothing
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt: (150, 61): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type ReadableMap?
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt: (151, 65): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type ReadableArray?
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt: (171, 69): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type ReadableMap?
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/BridgeHelpers.kt: (172, 73): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type ReadableArray?
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/ScanditModule.kt: (8, 90): Type mismatch: inferred type is ReactApplicationContext? but ReactApplicationContext was expected
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/ScanditPackage.kt: (10, 1): Class 'ScanditPackage' is not abstract and does not implement abstract member @Nonnull public abstract fun createNativeModules(@Nonnull p0: ReactApplicationContext): (Mutable)List<NativeModule!> defined in com.facebook.react.ReactPackage
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/ScanditPackage.kt: (12, 5): 'createNativeModules' overrides nothing
e: /projectpath/BugReport/node_modules/scandit-react-native/android/src/main/java/com/scandit/reactnative/ScanditPackage.kt: (18, 5): 'createViewManagers' overrides nothing
> Task :scandit-react-native:compileDebugKotlin FAILED
```

Repro: https://github.com/thymikee/scandit-bug-report
Steps to reproduce: 

```sh
yarn
yarn react-native run-android
```

**Comment by turnrye** [comment]
This is called out here in the changelog:

> Add nullable annotations to ReadableMap, WritableMap, ReadableArray, Writable, ReactPackage, and native module interfaces; this may impact Kotlin usage

https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#android-specific-1

Should be a quick fix.

**Comment by dadish** [comment]
Is there any time estimates on when this gonna be fixed. It's a blocker for our project at the moment.

**Comment by moritzhartmeier** [comment]
We have already invested time into fixing this at the end of last week and so far it has not been a super quick fix. We are currently in testing for 5.11 and are working on fixing this issue with that release as well. This means we are looking at a fix this or next week.

**Comment by moritzhartmeier** [comment]
Thanks to @TomekJurkowski and @petradonka this has been fixed with #112 #113 and #114 in 5.10 onwards.

**Comment by kroofy** [comment]
> Thanks to @TomekJurkowski and @petradonka this has been fixed with #112 #113 and #114 in 5.10 onwards.

Thanks for fixing the issue! 🥇 

Just to avoid confusion though. Since 5.10 is the current release. You probably meant to say 5.11, no?

**Comment by moritzhartmeier** [comment]
If you get it from npm then it will be available in 5.11, if you reference this github repo directly you can get the fix in the stable 5.10 branch already.
